ale_linters/purescript/purels.vim

```diff
@@ -0,0 +1,50 @@
+" Author: Drew Olson <drew@drewolson.org>
+" Description: Integrate ALE with purescript-language-server.
+
+call ale#Set('purescript_purels_executable', 'purescript-language-server')
+call ale#Set('purescript_purels_use_global', get(g:, 'ale_use_global_executables', 0))
+call ale#Set('purescript_purels_config', {})
```

Comment:
Name the file and the variables `ls` instead of `purels`. That will more closely match how other files are named.